src/base/dense/owl_base_dense_matrix_intf.ml

```diff
@@ -0,0 +1,20 @@
+(*
+ * OWL - OCaml Scientific and Engineering Computing
+ * Copyright (c) 2016-2019 Liang Wang <liang.wang@cl.cam.ac.uk>
+ *)
+
+
+module type Common = sig
```

Comment:
Did you use `Base` in previous intf? https://github.com/owlbarn/owl/blob/0fdf627a8633e0ee26ee3c44526e3e55aa67acca/src/owl/dense/owl_dense_ndarray_intf.ml#L9

Shall we stay consistent here? Or did I miss any obvious reason for not doing so?

Reply:
updated in the latest commit